index.html

```diff
@@ -10,6 +10,7 @@
     <svg xmlns="http://www.w3.org/2000/svg">
       <circle cx="50" cy="50" r="50" fill="red" />
     </svg>
-    
+
+    <script type="text/javascript" src="scripts/main.js"></script>    
   </body>
 </html>
```

Comment:
this little red arrow is saying I should add a new line to the file.  They should all have that at the end.  Something about C parsing files or something, it's weird, we should just do it.